docs/source/about/changelog.rst

```diff
@@ -69,6 +70,7 @@ Unreleased
 **Fixed**
 
 - :pull:`1239` - Fixed a bug where script elements would not render to the DOM as plain text.
+- :pull:`1271` - Fixed bug where JavaScript components were unable to obtain the ``key`` attribute provided within ReactPy.
```

Comment:
Can you double check that my changelog entry of this issue is characterized correctly?

Reply:
I tweaked the verbiage a bit, should be good to go.